R/get_genstate_covid.R

```diff
@@ -0,0 +1,85 @@
+#' Get general population state COVID data from CDC and Census 2020 pop estimates
+#'
+#' @description Pulls data from the CDC on COVID cases and deaths
+#' for a state and then merges alongside a population number from the 2020
+#' Census population estimates for that state to use for rate calculation.
+#'
+#' @return a data frame with the following columns: State, Date,
+#' General.Confirmed, General.Deaths, General.Population
+#'
+#' @examples
+#' \dontrun{
+#' state_ <- "Georgia"
+#'
+#'# get state level covid data
+#' st_gen_df <- get_genstate_covid() %>%
+#'     filter(State == state_)
```

Comment:
Added this to the vaccine PR but should've noticed here too – could we add just `get_genstate_covid()` as an example that gets run? Extremely likely nothing breaks, but I like the idea of the CI catching if a link or something changes.

Reply:
added!